### DISCUSSION TITLE
The DKIM record is not working.

### DISCUSSION THREAD

**Discussion by Acoustical** [postalserver/postal#1795] [Help with using Postal] [answered]
![dkim](https://user-images.githubusercontent.com/23265557/148207616-6c6d85e4-31f8-484f-b96d-401b4b39714c.png)
I've set up the DKIM record and used `nslookup` to check whether it works. But it says two records are not matched.

**Comment by willpower232** [comment, marked as answer]
You can confirm by running nslookup on the server itself. It might be that the server has weird or slow DNS.
